data_diff/database.py

```diff
@@ -852,6 +862,7 @@ class Snowflake(Database):
     NUMERIC_TYPES = {
         "NUMBER": Decimal,
         "FLOAT": Float,
+        "BIGINT": Decimal,
```

Comment:
I believe this should be `Integer`.

Did you add this because you got an error in the tests?